dist/Components/upperInfo/layout.jsx

```diff
@@ -55,15 +82,35 @@ class Layout extends React.Component {
   render() {
     return (
       <div>
-        <FontAwesomeIcon icon="home" />
-        <Type>Entire house</Type>
-        <Wrapper>
-          <Number>{this.state.guests} guests&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Number>
-          <Number>{this.state.rooms} bedrooms&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Number>
-          <Number>{this.state.beds} beds&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Number>
-          <Number>{this.state.baths} baths</Number>
-        </Wrapper>
-        <Line />
+        <div>
+          <FontAwesomeIcon icon="home" />
+          <Type>Entire house</Type>
+          <Wrapper>
+            <span>{this.state.guests} guests&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
```

Comment:
I am sure you have thought about adding some padding or fixed width here instead of these `&nbsp;`s!